Example/ConsentViewController_ExampleTests/SPClientCoordinator/SourcePointClient/UnmockedSourcepointClientSpec.swift

```diff
@@ -196,6 +196,28 @@ class UnmockedSourcepointClientSpec: QuickSpec {
             }
         }
 
+        describe("meta-data_groupPmId") {
+            it("Check if groupPmId echo") {
```

Comment:
Same goes here, since we already have a `describe("meta-data")` you don't need to duplicate it, you can simply add the new `it("...` to the existing `describe` a few lines above.